src/main/java/org/opentripplanner/netex/configure/NetexConfig.java

```diff
@@ -82,7 +82,8 @@ private NetexBundle netexBundle(ConfiguredDataSource<NetexFeedConfig> netexConfi
       (CompositeDataSource) netexConfiguredDataSource.dataSource(),
       hierarchy(netexConfiguredDataSource),
       buildParams.netexDefaults.ferryIdsNotAllowedForBicycle,
-      buildParams.maxStopToShapeSnapDistance
+      buildParams.maxStopToShapeSnapDistance,
+      buildParams.netexDefaults.noTransfersOnIsolatedStops
```

Comment:
Shouldn't this be read from `netexConfiguredDataSource.config()`, not the defaults?